index.html

```diff
@@ -9,15 +9,84 @@
   <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
-<body>
+<body class="padding-app">
   <h1 class="title">Front-End Tools</h1>
-  <div class="tech-box">
-    <img class="tech-box-logo" src="assets/tech/react_icon.svg" alt="Logo React"/>
-    <h2 class="tech-box-title">ReactJS</h2>
-    <span class="tech-box-launch">Release: 2013</span>
-    <p class="tech-box-text">
-      React is a declarative, component-based JavaScript library for building user interfaces.
-    </p>
+  <div class="tech-box-list">
+    <div class="tech-box">
+      <img class="tech-box-logo" src="assets/tech/react_icon.svg" alt="Logo React"/>
+      <h2 class="tech-box-title">ReactJS</h2>
+      <span class="tech-box-launch">Release: 2013</span>
+      <p class="tech-box-text">
+        React is a declarative, component-based JavaScript library for building user interfaces.
+      </p>
+    </div>
+    <div class="tech-box">
+      <img class="tech-box-logo" src="assets/tech/angular_icon.svg" alt="Logo AngularJS"/>
```

Comment:
Same than previous PR, in english the word `logo` should be after the name of the tech